recipes/libunwind/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set name = "libunwind" %}
+{% set version = "1.2.1" %}
+{% set sha256 = "3f3ecb90e28cbe53fba7a4a27ccce7aad188d3210bb1964a923a731a27a75acb" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: http://download.savannah.nongnu.org/releases/libunwind/libunwind-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [win or osx]
+
+requirements:
+  build:
+    - toolchain
+
+test:
+  commands:
+    - test -f ${PREFIX}/lib/libunwind.so
```

Comment:
Is there a header we should check for?

Reply:
Copied over a few additional checks from the `libosxunwind` recipe